docs/theory/apportion_abundance.md

```diff
@@ -113,115 +134,119 @@ $$
 
 #### Unaged fish
 
-The number counts of unaged fish across $\ell$ for each $s$ $\eqref{eq:total_unaged_sex_length}$ relative to the sex-specific totals $\eqref{eq:total_unaged_sex}$, $r_{n,s,\ell}^{\textrm{unaged/unaged}}$, are:
+The number proportions of male and female unaged fish of length $\ell$ $\eqref{eq:total_unaged_sex_length}$ relative to the sex-specific totals of unaged fish $\eqref{eq:total_unaged_sex}$ are:
 
 $$
 \begin{equation}
 \begin{aligned}
-    r_{n,~\textrm{M},\ell}^{\textrm{unaged/unaged}} &= \frac{n_{\textrm{M},\ell}^{\textrm{unaged}}}{n_{\textrm{M}}^{\textrm{unaged}}} \nonumber \\
-    r_{n,~\textrm{F},\ell}^{\textrm{unaged/unaged}} &= \frac{n_{\textrm{F},\ell}^{\textrm{unaged}}}{n_{\textrm{F}}^{\textrm{unaged}}} \nonumber    
+    {r_N}_{\textrm{M},\ell}^{\textrm{unaged/unaged}} &= \frac{n_{\textrm{M},\ell}^{\textrm{unaged}}}{n_{\textrm{M}}^{\textrm{unaged}}} \nonumber \\
```

Comment:
Here I think these should be $r_n$. To me this implies that these are relative proportions calculated from abundances when they are actually computed using the fish numbers/counts. Perhaps that is not a significant distinction given that these are used to apportion the derived abundances; however, I think if we use $r_N$ here then we would want to use $r_B$ (we currently do not have a subscript denoting "weight" specifically for this since we just use $r$ currently) for the weight proportions since they are used to apportion the derived biomasses.

Reply:
$r_n$ is fine - this is something I had not changed after changing all $N$ to $n$ in [#280](https://github.com/OSOceanAcoustics/echopop/pull/280)